.github/workflows/watch-z2jh-chart.yml

```diff
@@ -0,0 +1,53 @@
+name: Check for new versions of the pangeo/base-notebook image
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'pangeo-data/helm-chart'
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get latest JupyterHub tag
+        id: z2jh_tag
+        uses: jacobtomlinson/gha-get-helm-chart-tags@TODO
+        with:
+          chart: "https://jupyterhub.github.io/helm-chart/"
+      - name: Get latest Dask-Gateway tag
+        id: gateway_tag
+        uses: jacobtomlinson/gha-get-helm-chart-tags@TODO
+        with:
+          chart: "https://dask.org/dask-gateway-helm-repo/"
```

Comment:
This action (`gha-get-helm-chart-tags`) does not exist yet. Just sketching out some ideas here.

Reply:
🤣 I love this! I do plan on working on this soon(ish). We need it for the RAPIDS Helm chart.